data/colors/themes/light_tritanopia.ts

```diff
@@ -0,0 +1,62 @@
+import {merge} from '../../../src/utils'
+import light_colorblind from './light_colorblind'
```

Comment:
Is it ok to import `light_colorblind` directly (instead of `light`)? It would avoid having to keep the [colorblind exceptions](https://github.com/primer/primitives/blob/251de59997a90bb9500a87152fcbee99798ae3e8/data/colors/themes/light_colorblind.ts#L62-L93) in sync.